userguide/sharing.rst

```diff
@@ -1006,131 +1006,136 @@ for more details.
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | default_quota       | **Deprecated: use the ixnas module instead.** Store the default quotas that are reported to a Windows client in the quota       |
+   |                     | record of a user.                                                                                                               |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | dirsort             | Sort directory entries alphabetically before sending them to the client.                                                        |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | expand_msdfs        | Enable support for Microsoft Distributed File System (DFS).                                                                     |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | extd_audit          | Send :guilabel:`audit` logs to both syslog and the Samba log files.                                                             |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | fake_perms          | Allow roaming profile files and directories to be set as read-only.                                                             |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | fruit               | Enhance macOS support by providing the SMB2 AAPL extension and Netatalk interoperability.                                       |
+   |                     | Automatically loads *catia* and *streams_xattr* but read the caveat in NOTE below table.                                        |
```

Comment:
s/ but/, but/
", but see the warning below." (make "warning" a link to the note)